app/config/schema/rooms.proto

```diff
@@ -0,0 +1,171 @@
+/*
+ * This file defines and documents the format for rooms.json
+ *
+ * Proto doc: https://developers.google.com/protocol-buffers/
+ */
+
+syntax = "proto3"; // Must be proto3 to support JSON
+
+message Rooms {
+    repeated Room rooms = 1;
+}
+
+enum RoomType {
+    UNKNOWN = 0;
+    JITSI = 1;
+    IFRAME = 2;
+    CONTENT = 3;
+    IMAGEMAP = 4;
+    ART = 5;
+    REDIRECT = 6;
+}
+
+message Room {
+    // Unique identifier for the room within this house
+    // Used for defining paths between rooms and for creating Jitsi videos
+    // In loaded config, can be used to key into the rooms dict
+    // Required
+    string id = 1;
+
+    // User-facing display name of the room
+    // Required
+    string name = 2;
+
+    // What this room will contain
+    // Required
+    RoomType type = 3;
+
+    // User-facing description/flavor-text
+    // Optional
+    string description = 4;
+
+    // Relative filepath to an image to display behind room content
+    // Optional
+    string background_image = 5;
+
+    // Color for the entrance door, formatted as a hex code or HTML color name
+    // TODO: What is the default?
+    // Optional
+    string door_tint = 6;
```

Comment:
@morganecf @hndrewaall What's the default value for this? (I'm sure I could've figured this out on my own, but also, I was feeling lazy and I'm tired)

Reply:
[There doesn't appear to be one.](https://github.com/morganecf/jitsi-party/blob/master/app/client/jsx/Door.jsx#L38) Let's just set it to black as a default.

Reply:
Great! Set this to black and removed TODO comment.